packages/injecteo/pubspec.yaml

```diff
@@ -14,5 +14,5 @@ dependencies:
   get_it: ^7.2.0
 
 dev_dependencies:
-  lint: ^1.10.0
+  lint: ^2.0.1
```

Comment:
What do you think about moving from the `lint` package to the `linteo` instead?

Reply:
Done.